docs/.sphinx/rf_libraries/convert_libdoc_to_md.py

```diff
@@ -223,14 +229,49 @@ def main():
     repo_root = docs_dir.parent
 
     for target in RobotFile:
+        for source_pkg in PACKAGE[target]:
+            source_dir = repo_root / source_pkg
+            idx = source_dir.parts.index("rf_libraries")
+            
+            target_dir = docs_dir / Path("reference", *source_dir.parts[idx:])
+
+            if not target_dir.exists():
+                os.makedirs(target_dir)
+            
+            if target is RobotFile.LIBRARY and not (target_dir / "index.md").exists():
+                with open(target_dir / "index.md", "w") as index_file:
+                    platform_name = re.sub(r'_+', ' ', target_dir.stem.strip('_'))
+                    if platform_name == "libraries":
+                        platform_name = "base"
+
+                    index_file.write(f"# {platform_name.capitalize()} libraries\n\n")
+                    if platform_name == "base":
+                        index_file.write(
+                            dedent(
+                                """
+                                We provide the following base classes for implementing a new platform plugin.
```

Comment:
```suggestion
                                YARF provides the following base classes for implementing a new platform plugin.\n
```
I suppose you want them in two separate lines